cwd/echo/echo.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+)
+
+func main() {
+	arg := os.Args
+	var showLines bool
+	flag.BoolVar(&showLines, "n", false, "New line")
```

Comment:
is it omit new line or show new line

Reply:
for echo we print all of the args e.g try `echo arg1 arg2 arg3`  that will never work in your code